packages/dev/docs/src/docs.js

```diff
@@ -36,29 +37,56 @@ for (let link of links) {
         <Pressable>
           {/* eslint-disable jsx-a11y/click-events-have-key-events */}
           {/* eslint-disable jsx-a11y/anchor-is-valid */}
-          <a role="link" tabIndex={0} data-link={link.dataset.link} className={link.className} onClick={e => e.preventDefault()}>{link.textContent}</a>
+          <a role="button" tabIndex={0} aria-haspopup="dialog" data-link={link.dataset.link} className={link.className} onClick={e => e.preventDefault()}>{link.textContent}</a>
         </Pressable>
         <LinkPopover id={link.dataset.link} />
       </DialogTrigger>
     </ThemeProvider>
   , container);
 
-  link.parentNode.replaceChild(container, link);
+  link.parentNode.replaceChild(container.firstElementChild, link);
 }
 
 function LinkPopover({id}) {
   let ref = useRef();
   let [breadcrumbs, setBreadcrumbs] = useState([document.getElementById(id)]);
 
+  const onBlurCurrentBreadcrumb = useCallback((event) => {
+    event.target.removeEventListener('blur', onBlurCurrentBreadcrumb);
+    event.target.tabIndex = -1;
+  },
+  []);
+
   useEffect(() => {
+    let isCancelled = false; 
+    let timeoutId;
     let links = ref.current.querySelectorAll('[data-link]');
     for (let link of links) {
+      link.removeAttribute('aria-haspopup');
```

Comment:
I do not understand what is happening here, why is aria-haspopup removed?

Reply:
When a `Link` within a code sample appears within a `LinkPopover` it will no longer open in a new popover, but instead refreshes the contents of the current popover, so `aria-haspopup` is not really appropriate.